source/solvers/mf_navier_stokes.cc

```diff
@@ -44,6 +44,24 @@
 
 #include <deal.II/numerics/vector_tools.h>
 
+/**
+ * @brief Helper function that allows to convert deal.II vectors to Trilinos vectors.
+ *
+ * @tparam number Abstract type for number across the class (i.e., double).
+ * @param out Destination TrilinosWrappers::MPI::Vector vector.
+ * @param in Source LinearAlgebra::distributed::Vector<number> vector.
+ */
+template <typename number>
```

Comment:
Could this be moved outside of the MF files? I feel like this could be useful in many placed

Reply:
We could move it but I think we should leave it here for now... this is the only place where it is used and eventually we will remove it once multiphysics support also `LinearAlgebra::distributed::Vector`

Reply:
OKidoke